backend/src/core/config.ts

```diff
@@ -568,6 +571,13 @@ const config: Config<ConfigSchema> = convict({
       format: 'required-string',
       sensitive: true,
     },
+    callbackUrl: {
```

Comment:
**[request for more context]** curious about adding this (haven't traced the code fully myself), this is used for all callbacks and not just tx SMS callbacks right? how did we handle callbacks previously for campaign SMSes/why do we only need to add this now?

Reply:
ohh saw the changes at `backend/src/sms/routes/sms-callback.routes.ts` → seems like this is only used for tx SMS? if so should make clear in the `doc` field here. 

still wish to get more context on how this setup differs from campaign SMSes' callback though

Reply:
if we look over to the `worker` codebase, we'd notice that the variable that worker was referring to was the `backend_url` variable. this is simply explicitly labelling the `backend_url` correctly as the callback_url. 

If you scroll slightly up, you'd see that this is already nested under `smsCallback`, as it indicates that it is only used for smsCallbacks.

Reply:
Thanks for the clarification! Could you help to fix the `config.ts` [file here](https://github.com/opengovsg/postmangovsg/blob/a0ba537cb5908ae5c8eb5e554ce0313b023a3d79/worker/src/core/config.ts#L228-L233) so that it's aligned to these changes? So what happens in practice is the worker and backend actually are loaded with the same set of env variables, so it'd be good to keep them aligned.

Also can consider whether we should rename `backend_url` as it's specifically the SMS callback URL

Reply:
woah, ok would recommend not to change the worker file this release yet no matter how small, if this is really imported i will change it in another PR after the release. 

We probably should indeed rename `backend_url` but that would mean changing on secrets manager as well.